spec/jobs/signal_adapter/receive_polling_job_spec.rb

```diff
@@ -164,5 +165,31 @@
         end
       end
     end
+
+    describe 'given the Signal server is unavailable' do
+      let(:error_message) do
+        [['error', "Error while checking account #{Setting.signal_server_phone_number}: [502] Bad response: 502 \n"]].to_json
+      end
+
+      before do
+        create(:request)
+
+        unless Setting.signal_server_phone_number
+          allow(Setting).to receive(:signal_server_phone_number).and_return('SIGNAL_SERVER_PHONE_NUMBER')
+        end
```

Comment:
Why do you stub the setting only if it’s not set?

Reply:
It's a copy/:spaghetti: from the test setup on https://github.com/tactilenews/100eyes/blob/main/spec/jobs/signal_adapter/receive_polling_job_spec.rb#L44-L52, which was added by you. To be honest, I didn't really think too much about why you added it, I just moved the specs I added to their own block cause I didn't want to use the vcr, but I knew they were working with the test setup.

Reply:
I think my name’s in git blame only due to some reformatting/naming changes. Seems like Robert added this originally. This shouldn’t block the PR from getting merged, but maybe you could enlighten us why that’s necessary, @roschaefer? Thanks! :)